user/multi-cpu-architectures.md

```diff
@@ -147,31 +174,7 @@ You can try it out also for `ppc64le` (IBM Power) and `s390x` (IBM Z) based dock
 
 You can also have a look at [Using Docker in Builds](user/docker/).
 
-## Security and LXD Container
-
-### Access to Privileged fs/Features (Apparmor)
-
-> Due to security reasons, builds run in LXD containers will be denied access to privileged filesystems and paths - a privileged container with write access to e.g. /sys/kernel/debugfs might muddle an LXD host.
-
-As a result, for instance a command in `.travis.yaml` like:
-```yaml
-sudo docker run --privileged --rm -t -v /sys/kernel/debug:/sys/kernel/debug:rw
-```
-{: data-file=".travis.yml"}
-
-would result in an error.
-
-Also have a look at the [Github issue relevant to the topic](https://github.com/lxc/lxd/issues/2661) and the [LXD apparmor setup](https://github.com/lxc/lxd/blob/master/lxd/apparmor/apparmor.go) for more details.
-
-### System Calls Interception
-
-
-If you run into a message like:
-
-> System doesn't support syscall interception
-
-It most probably means a system call interception is outside of the list of the ones considered to be safe (LXD can allow system call interception [if it's considered to be safe](https://github.com/lxc/lxd/blob/master/doc/syscall-interception.md)). 
 
-## Hugepages Support from within LXD Container
+## LXD related limitations
 
-A build job can’t enable hugepages within an LXD container - this is something that may change in the future, yet it depends on potential Linux kernel changes, which is something that needs to be reviewed and developed.
+Pleae have a look at [Build Environment Overview](/user/reference/overview/#linux-security-and-lxd-container) for more details.
```

Comment:
For more details, see [Build Environment Overview](/user/reference/overview/#linux-security-and-lxd-container)

Also, there is a typo on the Please it currently reads "Pleae"

Reply:
Fixing on master, thanks!